examples/example/example.ino

```diff
@@ -37,7 +37,7 @@ void messageReceived() {
 void setup() {
     radio.init();
     radio.setSyncWord(syncWord);
-    radio.setCarrierFreq(CFREQ_433);
```

Comment:
Why change from 433MHz to 900 MHz in the example? 900 MHz is out of the ISM band in the ITU Region 2!

Background information:
- 433.05 MHz to 434.79 MHz is for ITU Region 1, which comprises Europe, Africa, the former Soviet Union, Mongolia, and the Middle East west of the Persian Gulf, including Iraq.
- 902 to 928 MHz is for ITU Region 2, which covers the Americas including Greenland, and some of the eastern Pacific Islands.